packages/axios-extension/test/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "../tsconfig.json",
+    "include": ["../../src/**/*.ts", "**/*.ts"],
+    "exclude": ["dist", "node_modules"],
+    "compilerOptions": {
+        "types": ["jest", "node"],
+        "baseUrl": ".",
+        "paths": { "@src/*": ["../src/*"] }
+    }
+}
```

Comment:
As one of the earlier comments, how does this file help and would it make sense to do it for all projects?

Reply:
In this case, it's exclusively used for path mapping. 
The imports look this now:
```typescript
import Foo from @src/foo;
```
instead of:
```typescript
import Foo from '../../src/foo';
```

It would make sense in other projects too, but we should probably do it organically

Reply:
It can be better for performance to transpile `src` and `test` separately too.
Maybe worth considering as part of https://github.com/SAP/open-ux-tools/issues/495